client/post-editor/post-editor.jsx

```diff
@@ -417,8 +417,8 @@ export const PostEditor = React.createClass( {
 				page.redirect( utils.getEditURL( post, site ) );
 			}
 			this.setState( postEditState, function() {
-				if ( didLoad || this.state.isLoadingAutosave ) {
-					this.refs.editor.setEditorContent( this.state.post.content, { initial: true } );
+				if ( this.editor && ( didLoad || this.state.isLoadingAutosave ) ) {
+					this.editor.setEditorContent( this.state.post.content, { initial: true } );
 				}
 
 				if ( this.state.isLoadingAutosave ) {
```

Comment:
> the chances for this to happen are really low I think.

well still great enough that it's collecting error reports. that's why I'm updating the code.

> But if this.editor is falsy this means, we'll probably have an error here as well, so maybe we can just shortcut the callback if this.editor is falsy?

I'll check in on this. The solution I wrote up is quick and simple. Maybe we have more robust refactoring in store.